astropy/io/ascii/ui.py

```diff
@@ -1059,10 +1058,14 @@ def write(
         # behavior is for Python to translate \r\n (which we write because
         # of os.linesep) into \r\r\n. Specifying newline='' disables any
         # auto-translation.
-        output = open(output, "w", newline="")
-        output.write(outstr)
-        output.write(os.linesep)
-        output.close()
```

Comment:
This should just be changed to an `open(...)` context manager.

Reply:
done